File: web/src/lib/categories.ts
export const CATEGORIES: { label: string; value?: string }[] = [
  { label: 'All' },
  { label: 'Responding to resistance', value: 'defiance' },
  { label: 'Engagement & participation', value: 'disengagement' },
  { label: 'Conflict & repair', value: 'peer_conflict' },
  { label: 'Interruptions & redirection', value: 'disruption' },
  { label: 'Routines & transitions', value: 'transitions' },
  { label: 'Devices & digital routines', value: 'technology_misuse' },
]

export function categoryLabel(value: string) {
  return CATEGORIES.find((c) => c.value === value)?.label ?? value
}

File: ClassCoach/Models/API/Categories.swift
import Foundation

/// Mirrors `web/src/lib/categories.ts` — shared by Try It Out and
/// Ask/Debrief, both of which tag content with the same category strings.
let scenarioCategories: [(label: String, value: String?)] = [
    ("All", nil),
    ("Defiance", "defiance"),
    ("Disengagement", "disengagement"),
    ("Peer conflict", "peer_conflict"),
    ("Disruption", "disruption"),
    ("Transitions", "transitions"),
    ("Technology misuse", "technology_misuse"),
]

func categoryLabel(_ value: String) -> String {
    scenarioCategories.first { $0.value == value }?.label ?? value
}
